main.tf

```
@@ -0,0 +1,117 @@
+resource "aws_vpc" "demo_vpc" {
+  cidr_block = "10.0.0.0/16"
+  tags = {
+    "Name" = "demo-vpc"
+  }
+}
+resource "aws_subnet" "public0" {
```

Top-level blocks should always be separated from one another by one blank line. 

```suggestion
}

resource "aws_subnet" "public0" {
```

---

Please adjust it here and for all other top-level blocks

---

ok